lib/Service/Appointments/BookingService.php

```diff
@@ -128,6 +151,14 @@ public function confirmBooking(Booking $booking, AppointmentConfig $config): Boo
 			$this->logger->warning('Could not send booking information notification after confirmation by user ' . $booking->getEmail(), ['exception' => $e]);
 		}
 
+
+		$this->eventDispatcher->dispatchTyped(
+			new AppointmentBookedEvent(
+				$booking,
+				$config
+			)
+		);
```

Comment:
:boom: or this one

Reply:
this because the listener sets the talk URL on the object passed through the event